backend-api/src/functions/adapters/kmsAdapter.ts

```diff
@@ -1,50 +1,70 @@
 import {
   DecryptCommand,
   DecryptCommandOutput,
+  IncorrectKeyException,
+  InvalidCiphertextException,
   KMSClient,
 } from "@aws-sdk/client-kms";
-import { NodeHttpHandler } from "@smithy/node-http-handler";
 import { errorResult, Result, successResult } from "../utils/result";
+import { NodeHttpHandler } from "@aws-sdk/node-http-handler";
```

Comment:
As we're using the aws sdk v3 I think the smithy package is the preferred option (what we had before). Both work of course!

Reply:
I didn't mean to change this, it was a mistake, sorry.